stacklet/client/platform/cli.py

```diff
@@ -210,9 +214,25 @@ def auto_configure(url, location):
             "auth_url": auth_url,
             "cubejs": f"https://{config['cubejs_domain']}",
         }
-        saml_config = config.get("saml")
-        if saml_config:
-            formatted_config["idp_id"], _ = saml_config.popitem()
+        if saml_config := config.get("saml"):
+            if len(saml_config) == 1:
+                idp_id, _ = saml_config.popitem()
+            else:
+                name_to_id = {idp_id: name for name, idp_id in saml_config.items()}
```

Comment:
This seems backwards to me, which makes it hard to understand when reading the code. Shouldn't it be:
```
name_to_id = {name: idp_id for idp_id, name in saml_config.items()}
```